From: "Barak A. Pearlmutter" <barak+git@cs.nuim.ie>
Date: Tue, 1 Nov 2011 12:36:57 +0000
Subject: typos

---
 README | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/README b/README
index 4595393..76ff9ad 100644
--- a/README
+++ b/README
@@ -6,7 +6,7 @@ script syntax, syntax-case, unicode strings, bytevectors,
 user-defined record types, exception handling, conditions, and
 enumerations.  Over 80% of the R6RS procedures and keywords are
 currently implemented and subsequent releases will proceed towards
-brining Ikarus to full R6RS conformance.
+bringing Ikarus to full R6RS conformance.
 
 The main purpose behind releasing Ikarus early is to give Scheme
 programmers the opportunity to experiment with the various new
